django_autoconfig/app_settings.py

```diff
@@ -5,6 +5,9 @@
 #: Extra URLs for autourlconf
 AUTOCONFIG_EXTRA_URLS = getattr(settings, 'AUTOCONFIG_EXTRA_URLS', ())
 
+#: A list/tuple of apps to exclude from the autourlconf
+AUTOCONFIG_EXCLUDE_URLS = getattr(settings, 'AUTOCONFIG_EXCLUDE_URLS', ())
```

Comment:
I'm not a fan of the name here, the one above is about adding URLs, this is about excluding apps from the urlconf
Maybe `AUTOCONFIG_URLCONF_EXCLUDE_APPS`?